docs/docs/configuration/overview.md

```diff
@@ -128,6 +128,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--skip-provider-button` | bool | will skip sign-in-page to directly reach the next step: oauth/start | false |
 | `--ssl-insecure-skip-verify` | bool | skip validation of certificates presented when using HTTPS providers | false |
 | `--ssl-upstream-insecure-skip-verify` | bool | skip validation of certificates presented when using HTTPS upstreams | false |
+| `--sensitive-logging` | bool | Log sensitive data like user email | false |
```

Comment:
I lean toward naming this something like `scrub-logs` -- using the term `scrub` for this operation is decently standard from what I've seen.

This also should not scrub sensitive fields by default, I think that should be an opt-in flag since this isn't a concern to a majority of users and they likely want the emails for debugging if they don't have strict PII regulations forcing them to scrub.